code/modules/admin/topic.dm

```diff
@@ -4583,3 +4584,232 @@
+
+				switch(book_style)
+					if("Koran")
+						rel_added.holy_book.icon_state = "koran"
+						rel_added.holy_book.item_state = "koran"
+					if("Scrapbook")
+						rel_added.holy_book.icon_state = "scrapbook"
+						rel_added.holy_book.item_state = "scrapbook"
+					if("Creeper")
+						rel_added.holy_book.icon_state = "creeper"
+						rel_added.holy_book.item_state = "syringe_kit"
+					if("White Bible")
+						rel_added.holy_book.icon_state = "white"
+						rel_added.holy_book.item_state = "syringe_kit"
+					if("Holy Light")
+						rel_added.holy_book.icon_state = "holylight"
+						rel_added.holy_book.item_state = "syringe_kit"
+					if("Athiest")
+						rel_added.holy_book.icon_state = "athiest"
+						rel_added.holy_book.item_state = "syringe_kit"
+					if("Tome")
+						rel_added.holy_book.icon_state = "tome"
+						rel_added.holy_book.item_state = "syringe_kit"
+					if("The King in Yellow")
+						rel_added.holy_book.icon_state = "kingyellow"
+						rel_added.holy_book.item_state = "kingyellow"
+					if("Ithaqua")
+						rel_added.holy_book.icon_state = "ithaqua"
+						rel_added.holy_book.item_state = "ithaqua"
+					if("Scientology")
+						rel_added.holy_book.icon_state = "scientology"
+						rel_added.holy_book.item_state = "scientology"
+					if("the bible melts")
+						rel_added.holy_book.icon_state = "melted"
+						rel_added.holy_book.item_state = "melted"
+					if("Unaussprechlichen Kulten")
+						rel_added.holy_book.icon_state = "kulten"
+						rel_added.holy_book.item_state = "kulten"
+					if("Necronomicon")
+						rel_added.holy_book.icon_state = "necronomicon"
+						rel_added.holy_book.item_state = "necronomicon"
+					if("Book of Shadows")
+						rel_added.holy_book.icon_state = "shadows"
+						rel_added.holy_book.item_state = "shadows"
+					if("Torah")
+						rel_added.holy_book.icon_state = "torah"
+						rel_added.holy_book.item_state = "torah"
+					if("Burning")
+						rel_added.holy_book.icon_state = "burning"
+						rel_added.holy_book.item_state = "syringe_kit"
+					if("Honk")
+						rel_added.holy_book.icon_state = "honkbook"
+						rel_added.holy_book.item_state = "honkbook"
+					if("Ianism")
+						rel_added.holy_book.icon_state = "ianism"
+						rel_added.holy_book.item_state = "ianism"
+					if("The Guide")
+						rel_added.holy_book.icon_state = "guide"
+						rel_added.holy_book.item_state = "guide"
+					if("Slab")
+						rel_added.holy_book.icon_state = "slab"
+						rel_added.holy_book.item_state = "slab"
+						rel_added.holy_book.desc = "A bizarre, ticking device... That looks broken."
+					else
+						//If christian bible, revert to default
+						rel_added.holy_book.icon_state = "bible"
+						rel_added.holy_book.item_state = "bible"
+
+				var/msg = "[usr] created a religion: [rel_added.name]."
+				msg = "<span class=\"admin\"><span class=\"prefix\">ADMIN LOG:</span> <span class=\"message\">[msg]</span></span>"
```

Comment:
why not `message_admins`? If you change this, change the other occurrences of this as well.